content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,57 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers:
+
+**From a user to a controller**: 
+
+First the user signals to the controller that it should take ownership of a
+field, usually by setting some other field that the controller observes. The
+controller then claims ownership of the field by applying a configuration
+containing the field. This can be done in one of the following ways:
+
+- If the controller is designed to immediately set the value of the field, the
+controller should apply with force set to true. Once the user observes that the
+controller has taken ownership of the field, they can remove the field from
+their applied configuration to complete the transfer.
+- If the controller will not set the value of the field immediately, the user
+should genereate a unique, temporary, field manager name, and transfer ownership
+of the field to it, similarly to the "user to user" instructions above. This transfer should be done by first setting the field to
+the same value in the applied configrations of the user and temporary field
+manager, and then by removing it from the user's applied configuration after
+field ownership is shared. Later, when the controller does want to set the value
+of the field, it can then apply with force set to true, which will automatically
+delete the temporary field manager without the need for any clean up.
+
+Transferring fields to controllers this way ensures the field is not accidentally
+removed from the live object, or reset to a default value, at any point of time
+during the transfer.
+
+This approach can be used, for example, to safely transfer ownership of the
+`replicas` field from a user to a controller while enabling Horizontal Pod
+Autoscaling on an object. Controllers should use apply, not update, to claim
+ownership of fields since only apply is able to claim ownership of a field even
+if the desired value is the same as the live value.
+
+**From a user to another user**: 
```

Comment:
Maybe actually move this section? :)

Reply:
Sounds good. I've made this a distinct markdown section, so the main section now focuses just on user->controller transfer.